
#include <Exception>

void archiveCheck() throw(std::exception) {
	throw std::exception();
}



